proto/secret/compute/v1beta1/query.proto

```diff
@@ -34,10 +34,24 @@ service Query {
     rpc Codes (google.protobuf.Empty) returns (QueryCodesResponse) {
         option (google.api.http).get = "/compute/v1beta1/code";
     }
+    // Query contract hash by address
+    rpc ContractHash (QueryContractHashRequest) returns (QueryContractHashResponse) {
+        option (google.api.http).get = "/compute/v1beta1/hash/address/{address}";
+    }
     // Query contract hash by code id
     rpc ContractHashByID (QueryContractHashByIDRequest) returns (QueryContractHashResponse) {
         option (google.api.http).get = "/compute/v1beta1/hash/id/{code_id}";
     }
+    // Query contract key by address
+    rpc ContractKey(QueryContractKeyRequest) returns (QueryContractKeyResponse) {
```

Comment:
Irrelevant here

Reply:
Why? it was implemented in the querier but was missed here, If it's irrelevant ill remove it from the querier as well.

Reply:
Because it's used only inside the enclave, there's no reason to use it outside